.gitlab-ci.yml

```diff
@@ -19,6 +19,21 @@ cppcheck:
   tags:
     - docker
 
+flake8:
+  stage: build
+  image: pipelinecomponents/flake8:latest
+  variables:
+      FLAKE8_FILES: "tests/capi.py"
+  script:
+    - flake8 --verbose $FLAKE8_FILES
```

Comment:
Hi,

Just a note: there is a trick with "git ls-files" to get list list of files under source control and has needed extension
   See more here: https://github.com/koalaman/shellcheck/wiki/GitLab-CI

That will allow not hard-code Python files paths in CI related file

Reply:
Did you read the commit message? The reason to pass them explicitly is that most python files currently don't pass flake8, and we want to check only the ones that pass already.

Reply:
Yes, i've read.
I've no objections for this PR.
My comment was not an issue for current PR but just a note which possibly you know to what future task to add.